packs/spells/purifying-icicle.json

```diff
@@ -69,7 +58,18 @@
             "value": "60 feet"
         },
         "requirements": "",
-        "rules": [],
+        "rules": [
+            {
+                "damageType": "vitality",
+                "diceNumber": "@spell.level",
+                "dieSize": "d4",
+                "key": "DamageDice",
+                "predicate": [
+                    "target:mode:undead"
```

Comment:
```suggestion
                    "target:negative-healing"
```

Reply:
The description says "...if the target is undead...". It's possible to have negative healing but not be undead?

Reply:
Are you going to change heal and harm, too?